pkgs/top-level/python-packages.nix

```diff
@@ -2606,6 +2592,27 @@ let
        license = licenses.mit;
      };
   };
+  
+  pytestrunner = buildPythonPackage rec {
+    version = "2.6.2";
+    name = "pytest-runner-${version}";
+    
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pytest-runner/${name}.tar.gz";
+      sha256 = "e775a40ee4a3a1d45018b199c44cc20bbe7f3df2dc8882f61465bb4141c78cdb";
+    };
+    
+    buildInputs = with self; [setuptools_scm pytest];
```

Comment:
The nix style seems to be to have a space between `[` and content. I.e. `[ setuptools_scm pytest ]` (and in all other locations)